data_model/1.4/clusters/BallastConfiguration.xml

```diff
@@ -118,37 +126,37 @@ Davis, CA 95616, USA
     </attribute>
     <attribute id="0x0014" name="IntrinsicBallastFactor" type="uint8">
       <access read="true" write="true" readPrivilege="view" writePrivilege="manage"/>
-      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false" sourceAttribution="false" quieterReporting="false"/>
+      <quality nullable="true"/>
       <optionalConform/>
     </attribute>
     <attribute id="0x0015" name="BallastFactorAdjustment" type="uint8" default="null">
       <access read="true" write="true" readPrivilege="view" writePrivilege="manage"/>
-      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false" sourceAttribution="false" quieterReporting="false"/>
+      <quality nullable="true"/>
       <optionalConform/>
       <constraint type="between" from="100" to="MS"/>
     </attribute>
     <attribute id="0x0020" name="LampQuantity" type="uint8">
       <access read="true" readPrivilege="view"/>
       <mandatoryConform/>
     </attribute>
-    <attribute id="0x0030" name="LampType" type="string" default="&quot;">
+    <attribute id="0x0030" name="LampType" type="string" default="&quot;&quot;">
```

Comment:
I assume this is due to the spec having "" in the default column, right?

Reply:
yeah, but it also doesn't matter currently because we don't handle defaults in the automated testing.